src/components/Tutorial/TutorialNavigation.tsx

```diff
@@ -20,7 +21,7 @@ import TutorialNavigationNext from './TutorialNavigationNext'
 import TutorialNavigationPrev from './TutorialNavigationPrev'
 
 // eslint-disable-next-line jsdoc/require-jsdoc
-const TutorialNavigation = ({ tutorialStep }: { tutorialStep: number }) => {
+const TutorialNavigation = ({ tutorialStep, nextRef }: { tutorialStep: number; nextRef: Ref<HTMLAnchorElement> }) => {
```

Comment:
Let's add a JSDOC comment for `nextRef` explaining its purpose.

Reply:
Added the comment here: e2fbc16f6408d1397a80f37d44a58c546a933add